apps/examples/App.tsx

```diff
@@ -107,6 +112,10 @@ const SCREENS: Record<
   },
 };
 
+const screens = Object.keys(SCREENS).filter(
+  name => !Platform.isTV || !!SCREENS[name].isTVOSready
```

Comment:
I wonder if it wouldn't be clearer to just do some method that returns check for each platform. What do you think?

Reply:
Good point, looks a little cleaner now:
https://github.com/software-mansion/react-native-screens/pull/2173/commits/1e5856ebcafa28b580d97515df082694772496ee